doc/classes/AudioEffectEQ.xml

```diff
@@ -1,8 +1,12 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="AudioEffectEQ" inherits="AudioEffect" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		Base class for audio equalizers. Gives you control over frequencies.
+		Use it to create a custom equalizer if [AudioEffectEQ6], [AudioEffectEQ10] or [AudioEffectEQ21] don't fit your needs.
 	</brief_description>
 	<description>
+		Equalizers gives you control over frequencies. Use it to compense existing deficiencies.
```

Comment:
It's ok to repeat the name: "AudioEffectEQ gives you control".

The second sentence I don't get.  Did you mean something like "Use it to compensate for existing deficiencies in audio." ???